.github/workflows/soak-testing.yml

```diff
@@ -0,0 +1,233 @@
+# Pre-requisites:
+# - AWS Account with correct permissions
+# - `gh-pages` branch
+# - LOGS_NAMESPACE? + LOGS_GROUP
+
+name: Soak Testing
+on:
+  schedule:
+    - cron: '0 */6 * * *' # Every 6 hours
+  # push:
+  #   branches: [ main ]
+env:
+  # Language Specific
+  PROCESS_COMMAND_LINE: /usr/local/bin/python3 application.py
+  PROCESS_EXECUTABLE_NAME: python3
+  # - Every language must also set the path for the sample-app in the `Build and Push Docker image` step.
+  # Global Env Vars
+  AWS_REGION: us-west-2
+  SOAK_TEST_DURATION_IN_MINUTES: 300
+  HOSTMETRICS_COLLECTION_INTERVAL_IN_SECONDS: 600
+  CPU_LOAD_THRESHOLD: 75
+  TOTAL_MEMORY_THRESHOLD: 1610612736
+  # Add this?
+  # RATE: TPS
```

Comment:
What's this?

Reply:
We were hoping to adapt these "Soak tests" for "Overhead tests" that could be run over a short duration (and possibly on every PR). They would [produce results like these for the Collector](https://github.com/aws-observability/aws-otel-collector/blob/main/docs/performance_model.md). But we would need our Load Generator to send requests at a certain `TPS` (which may required a different tool than `hey`) so I left this note as a reminder. I will update this note to be more clear about it.